python-sdk/tests/benchmark/infrastructure/kubernetes/create-nfs-server-gke.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
```

Comment:
@sunank200 What do you think of having all the NFS-related specs in the same file `nfs.yml`?
Since this is a declaration of infrastructure, I believe the file's name should be a resource, as opposed to action (eg. create). And it could simplify the maintenance t keep them all together.

Reply:
+1. Kubernetes is declarative, so you think in terms of nouns instead of verbs. Think "this is what I expect to exist" instead of "let me create this"

Reply:
Added it here: https://github.com/astronomer/astro-sdk/pull/945/commits/efff09b2e7d690f25968e15a1db2466f79c60521